pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -354,12 +357,20 @@ func (admitter *VMsAdmitter) authorizeVirtualMachineSpec(ar *admissionv1.Admissi
 			}
 		}
 
-		proxy := &authProxy{client: admitter.VirtClient, dataSourceInformer: admitter.DataSourceInformer, namespaceInformer: admitter.NamespaceInformer}
-		dv := &cdiv1.DataVolume{
+		dv, err := storagetypes.GetDataVolumeFromCache(targetNamespace, dataVolume.Name, admitter.DataVolumeInformer.GetStore())
+		if err != nil {
+			return nil, err
+		}
+		if dv != nil {
+			continue
```

Comment:
oh okay so you're skipping auth if dv exists. you could have also passed those to cloneAuthFunc if you wanted but not a must

Reply:
passing to the clone auth func would require that the dv have `prePopulated` annotation and I think that is an unnecessary requirement